README.MD

```diff
@@ -1,8 +1,295 @@
-# { json:api }
+# { json:api } Client
 
+[![PHP from Packagist](https://img.shields.io/packagist/php-v/swisnl/jsonapi.svg)](https://packagist.org/packages/swisnl/jsonapi)
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/swisnl/jsonapi.svg)](https://packagist.org/packages/swisnl/jsonapi)
+[![Software License](https://img.shields.io/packagist/l/swisnl/jsonapi.svg)](LICENSE)
 [![Run Status](https://api.shippable.com/projects/59ae7fe68a02980600016631/badge?branch=master)](https://app.shippable.com/github/swisnl/jsonapi)
 [![Coverage Badge](https://api.shippable.com/projects/59ae7fe68a02980600016631/coverageBadge?branch=master)](https://app.shippable.com/github/swisnl/jsonapi) 
 
-# WORK IN PROGRESS
+A PHP package for mapping remote [{json:api}](http://jsonapi.org/) resources to Eloquent like models and collections.
 
-A PHP package for mapping remote {json:api} resources to Eloquent like models and Collections.
\ No newline at end of file
+
+## Installation
+
+``` bash
+composer require swisnl/jsonapi
```

Comment:
Should this perhaps include a http adapter?

Reply:
I've added a separate section about the HTTP client just under this section.